torchvision/models/quantization/mobilenetv3.py

```diff
@@ -194,18 +194,33 @@ def mobilenet_v3_large(
     **kwargs: Any,
 ) -> QuantizableMobileNetV3:
     """
-    Constructs a MobileNetV3 Large architecture from
-    `"Searching for MobileNetV3" <https://arxiv.org/abs/1905.02244>`_.
+    MobileNetV3 model from
+    `Searching for MobileNetV3 <https://arxiv.org/abs/1905.02244>`_.
 
-    Note that quantize = True returns a quantized model with 8 bit
-    weights. Quantized models only support inference and run on CPUs.
-    GPU inference is not yet supported
+    .. note::
+        Note that ``quantize = True`` returns a quantized model with 8 bit
+        weights. Quantized models only support inference and run on CPUs.
+        GPU inference is not yet supported
```

Comment:
Thanks for making this a `note`, I agree this looks better this way. Would you like to open an issue to address the rest of the models? Or perhaps you'd like to submit a PR directly?

Reply:
On my way to submit PR :)